static/app/utils/theme/theme.chonk.tsx

```diff
@@ -0,0 +1,187 @@
+import type {SentryTheme} from './theme';
+
+const chonkShared = {
+  space: {
+    // @TODO(jonasbadalic): none doesn't need to exist
+    // none: 0,
+    subMicroscoptic: 1,
+    microscoptic: 2,
+    mini: 4,
+    small: 6,
+    medium: 8,
+    large: 12,
+    huge: 16,
+  },
+  borderRadius: {
+    // @TODO(jonasbadalic): none doesn't need to exist
+    // none: 0,
```

Comment:
why do you think `none` shouldn’t exist? Is it not a good thing to be able to overwrite a border-radius to zero ?

Reply:
I want to discourage overrides, and so by not having them on the theme, I am assuming it would make it less likely for someone to think they can use them as an actual value.

Basically, I wouldn't want someone to do styled(Button) with theme.radius.none thinking that that is a good use-case whereas that values really exists just for overrides - if they need overrides, they should do `border-radius: 0`